src/actions/blockchain.js

```diff
@@ -105,7 +113,7 @@ export const initGnosis = () => async (dispatch, getState) => {
       await getCurrentBalance(account)
     }
     await Promise.race([getConnection(), timeoutCondition(NETWORK_TIMEOUT, 'connection timed out')])
-    return await dispatch(setConnectionStatus({ connected: true }))
+    await dispatch(setConnectionStatus({ connected: true }))
```

Comment:
I wanted to ask, why are we awaiting for dispatch? AFAIK dispatch is synchronous

Reply:
Sometimes they're async because they're thunk actions. I want to be consistent, if I ever miss the `await` infront of a thunk action, it's going to be a very nasty bug